### PR TITLE
QPPSF-7909 - 2021 Benchmark Fix/Updates

### PR DESCRIPTION
**Pull request by shadford**
2021 Benchmark Fix/Updates

#### Motivation for change

- Fixed Issue with decile order
- Updated formats in FS benchmark generator to match QPP Measures Benchmark Format
- Fixed measure output ordering issues

#### What is being changed

Detailed information about what is being changed and rationale for interesting decisions made

#### Release checklist:
Tasks that must be done prior to merging this PR, including testing.

* [ ] [Package version updated](https://github.com/CMSgov/qpp-measures-data/blob/master/CONTRIBUTING.md#versioning-publishing-and-creating-new-releases)
* [ ] Documentation updated
* [ ] Unit tests added/passing
* [ ] Verified working locally

##### Associated JIRA tickets:
*  https://jira.cms.gov/browse/QPPSF-7909

### CODE REVIEW

File: benchmarks/2021.json
Comment (pkasarski) [inline benchmarks/2021.json:59]: Are these all exactly multiples of ten, except one, when last year they were all to two decimal places?

Comment (shadford) [reply benchmarks/2021.json:59]: this one jumped out as well, it's only happening with 001 too...

Comment (kevinburch-sb) [reply benchmarks/2021.json:59]: This is because 001 is a flat benchmark. I'd expect to see this behavior for 236 as well.

Comment (shadford) [reply benchmarks/2021.json:59]: Yes you are correct! I forgot about the Flats!

**Review by KyleApfel — CHANGES_REQUESTED** [review]
Not to be merged without administrative approval. Pending discussion.

**Review by KyleApfel — APPROVED** [review]
Given all clear to wrap this up in a new release. Thanks @shadford !

**Review by pkasarski — APPROVED** [review]

**Review by NateTheGreatt — APPROVED** [review]

**Review by pkasarski — CHANGES_REQUESTED** [review]
Need final approval from Steven.

**Comment by shadford** [comment]
Holding off publication until PI 14.3 targeting around August 2nd

**Review by KyleApfel — APPROVED** [review]
Clear to merge.

**Review by pkasarski — APPROVED** [review]

**Pull request merged**
